packages/charts/src/renderers/accessors/xy-accessors.ts

```diff
@@ -8,11 +10,35 @@ export interface IXYDataAccessors {
     /** Additional custom keys to match the base interface */
     [key: string]: DataAccessor | undefined;
 }
+export interface IXYSeriesAccessors {
+    /** Color of the series */
+    color?: SeriesAccessor;
+    /** Marker for the series */
+    marker?: SeriesAccessor;
+    /** Additional custom keys to match the base interface */
+    [key: string]: SeriesAccessor | undefined;
+}
+export interface IXYAccessors extends IAccessors {
+    data: IXYDataAccessors;
+    /** Series level accessors - e.g. for colors, markers, etc. */
+    series: IXYSeriesAccessors;
+}
 
-export abstract class XYAccessors implements IAccessors {
+export class XYAccessors implements IAccessors {
     /** The default data accessors for using with renderers deriving from XYRenderer */
     public data: IXYDataAccessors = {
         x: (d: any) => d.x,
         y: (d: any) => d.y,
     };
+    public series: IXYSeriesAccessors;
+
+    constructor(public colorProvider = defaultColorProvider(), public markerProvider = defaultMarkerProvider()) {
+
+        this.data.defined = (d: any, i) => d.hasOwnProperty("defined") ? d.defined : true;
+
+        this.series = {
+            color: this.colorProvider ? this.colorProvider.get : undefined,
+            marker: this.markerProvider ? this.markerProvider.get : undefined,
+        };
+    }
```

Comment:
I'm wondering if we need the color and marker accessors at all since we're not drawing anything on the chart.